e2e/pages/SignIn.js

```diff
@@ -59,6 +59,7 @@ class SignIn extends Page {
      * @param userName to login with
      * @param password to login with
      */
+    //TODO join with loginAs method
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)